docs/core/install/linux-ubuntu.md

```diff
@@ -35,20 +36,69 @@ The following table is a list of currently supported .NET releases and the versi
 
+## 22.10
+
+> [!WARNING]
+> If you've previously installed .NET from `packages.microsoft.com`, you may run into issues swapping to the built in Ubuntu package manager feeds for .NET. For more information, see the [Troubleshoot .NET errors related to missing files on Linux](linux-package-mixup.md),
+
+.NET 7 and .NET 6 are included in the Ubuntu 22.10 package manager feeds.
+
+### Install the SDK
+
+The .NET SDK allows you to develop apps with .NET. If you install the .NET SDK, you don't need to install the corresponding runtime. To install the .NET SDK, run the following commands:
+
+```bash
+sudo apt-get update && \
+  sudo apt-get install -y dotnet7
+```
+
+### Install the runtime
+
+The ASP.NET Core Runtime allows you to run apps that were made with .NET that didn't provide the runtime. The following commands install the ASP.NET Core Runtime, which is the most compatible runtime for .NET. In your terminal, run the following commands:
+
+```bash
+sudo apt-get update && \
+  sudo apt-get install -y aspnetcore-runtime-7.0
+```
+
+As an alternative to the ASP.NET Core Runtime, you can install the .NET Runtime, which doesn't include ASP.NET Core support: replace `aspnetcore-runtime-7.0` in the previous command with `dotnet-runtime-7.0`:
+
+```bash
+sudo apt-get install -y dotnet-runtime-7.0
+```
+
+## 22.10 (Microsoft package feed)
+
+> [!WARNING]
+> If you've previously installed .NET from `packages.microsoft.com`, you may run into issues swapping to the built in Ubuntu package manager feeds for .NET. For more information, see the [Troubleshoot .NET errors related to missing files on Linux](linux-package-mixup.md),
+
+[!INCLUDE [linux-prep-intro-apt](includes/linux-prep-intro-apt.md)]
+
+```bash
+wget https://packages.microsoft.com/config/ubuntu/22.10/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
+sudo dpkg -i packages-microsoft-prod.deb
+rm packages-microsoft-prod.deb
+```
+
+[!INCLUDE [linux-apt-install-70](includes/linux-install-70-apt.md)]
+
+> [!NOTE]
+> [Ubuntu 22.10 includes OpenSSL 3](https://discourse.ubuntu.com/t/openssl-3-0-transition-plans/24453) as the baseline version. Versions of .NET prior to .NET 6 don't support OpenSSL 3. Microsoft doesn't test or support using OpenSSL 1.x on Ubuntu 22.10. For more information, see [.NET 6 Security Improvements](https://devblogs.microsoft.com/dotnet/announcing-net-6/#security).
+
 ## 22.04
 
 > [!WARNING]
-> If you've previously installed .NET 6 from `packages.microsoft.com`, see the [Advisory on installing .NET 6 on Ubuntu 22.04](https://github.com/dotnet/core/issues/7699).
+> If you've previously installed .NET from `packages.microsoft.com`, you may run into issues swapping to the built in Ubuntu package manager feeds for .NET. For more information, see the [Advisory on installing .NET on Ubuntu](https://github.com/dotnet/core/issues/7699).
 
-.NET 6 is included in the Ubuntu 22.04 package manager feeds.
+.NET 7 and .NET 6 are included in the Ubuntu 22.04 package manager feeds.
```

Comment:
Only .NET 6 is included in Ubuntu 22.04 universe repo.

Reply:
.NET 7 isn't going to be published?

Reply:
is there an issue or something else to follow progress about when .NET 7 would be available on Ubuntu feeds, if ever?

Reply:
We can use this issue to track that: https://github.com/dotnet/core/issues/7939. We can update it as supports gets added. At present, .NET 7 isn't yet in any Ubuntu feed. As @mirespace suggests, .NET 7 is planned for 22.10+.

Reply:
So I guess the manual way of adding MS repos and installing from there is the way to go. It's fine, I was just confused whether I should expect the recently announced `apt install` experience to work for Ubuntu LTS.

Reply:
Understood. Why I wrote [the post](https://devblogs.microsoft.com/dotnet/dotnet-6-is-now-in-ubuntu-2204/) on that, I didn't know that this would be the plan.

For users that got with LTS only, then this approach will work well. For folks that are on the bleeding edge for everything, it will equally work well. For folks that want LTS Ubuntu + LTS/STS .NET, then it's a pain. In that case, using the MS feeds makes sense.

Separately, .NET 7 isn't yet ready for 22.10. That's just growing pains. The Canonical folks did a great job building and incorporating .NET 6. We (MS) did a poor job of walking Canonical folks through how to manage side-by-side with .NET versions. The delay is due to that. Once we get over the hurdle, we don't have the challenge again.